libs/pages/database/src/lib/ui/page-settings-general/page-settings-general.tsx

```diff
@@ -180,11 +175,15 @@ export function PageSettingsGeneral({
                   >
                     Learn more
                   </ExternalLink>
-                </>
+                </div>
               )}
             />
-          </BlockContent>
-          <div className="flex justify-end">
+          </Section>
+          <Section className="gap-4 mt-10">
```

Comment:
use `space-y-10` on `<form>` instead of `mt-10` everywhere

Reply:
https://github.com/Qovery/console/pull/1271/commits/6ed2bb30c904bff048362c528c5230e7bec1aff4

Reply:
This particular file has been missed

Reply:
https://github.com/Qovery/console/pull/1271/commits/d6276bb54e47bd564db78a461e9e03f76350a374